src/Services/GTT/shared/GTT.Infrastructure/Repositories/SportsRepository.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Data;
+using Dapper;
+using GTT.Application.Interfaces.Repositories;
+using GTT.Application.Requests;
+using GTT.Application;
+using GTT.Domain.Entities;
+using GTT.Application.Extensions;
+
+namespace GTT.Infrastructure.Repositories
+{
+    public class SportsRepository : ISportsRepository
+    {
+        #region Private Members
+        private readonly IDbConnection _connection;
+        #endregion
+
+        #region Constructors
+        public SportsRepository(IDbConnectionFactory dbConnectionFactory)
+        {
+            _connection = dbConnectionFactory.CreateConnection();
+        }
+
+        public SportsRepository(IDbConnection connection)
+        {
+            _connection = connection;
+        }
+        #endregion
+
+        public async Task<int> CreateSports(CreateSportRequestModel request)
```

Comment:
- Change function name to CreateSport
- Missing fields: CreatedAt, CreatedBy
- Table should be inclued UpdatedAt and UpdatedBy